app/controllers/events_controller.rb

```diff
@@ -10,7 +10,7 @@ def new
   def create
     @event = Event.new(event_params)
 
-    @event.user_id = session[:user_id]
+    @event.user_id = current_user.id
```

Comment:
We can use relation of User with events so that we don't have to do this, activerecord will auto add user_id
if we do this.
`current_user.events.new(event_params)` or use `build` as we are using for sponsors